### PR TITLE
CANCELED order status fixed parse bingx.py

### PR DESCRIPTION
**Pull request by ZetZ1**
BingX order status canceled really returns "CANCELLED", not CANCELED. Fixed parse

### CODE REVIEW

**Comment by carlosmiei** [comment]
@ZetZ1 Hello, actually I have seen it return 'CANCELED' as well, so is better to keep both.

**Comment by ZetZ1** [comment]
sorry, this status appears only for triggerorders i think

        #
        #    {
        #        "code": 0,
        #        "msg": "",
        #        "data": {
        #          "orders": [
        #            {
        #              "symbol": "LINK-USDT",
        #              "orderId": 1585839271162413056,
        #              "side": "BUY",
        #              "positionSide": "LONG",
        #              "type": "TRIGGER_MARKET",
        #              "origQty": "5.0",
        #              "price": "9",
        #              "executedQty": "0.0",
        #              "avgPrice": "0",
        #              "cumQuote": "0",
        #              "stopPrice": "5",
        #              "profit": "0.0000",
        #              "commission": "0.000000",
        #              "status": "CANCELLED",
        #              "time": 1667631605000,
        #              "updateTime": 1667631605000
        #            },
        #          ]
        #        }
        #    }

File: python/ccxt/bingx.py [hunk 1]
```diff
@@ -1124,19 +1124,13 @@ def fetch_balance(self, params={}):
         query for balance and get the amount of funds available for trading or funds locked in orders
         see https://bingx-api.github.io/docs/#/spot/trade-api.html#Query%20Assets
         see https://bingx-api.github.io/docs/#/swapV2/account-api.html#Get%20Perpetual%20Swap%20Account%20Asset%20Information
-        see https://bingx-api.github.io/docs/#/standard/contract-interface.html#Query%20standard%20contract%20balance
```
Comment (carlosmiei) [inline python/ccxt/bingx.py:1127]: @ZetZ1 can you please try to revert this file? We should only edit the TS one which I already did but for some reason can't restore this file.

**Review by ZetZ1 — COMMENTED** [review]
Revert file

**Comment by ZetZ1** [comment]
sorry, is my first "contribution in github, im not familiarized with procedure

its done?

**Comment by carlosmiei** [comment]
> sorry, is my first "contribution in github, im not familiarized with procedure
> 
> its done?

@ZetZ1 don't worry I managed to do it after all, will be merged soon.
Thanks once again for your contribution.

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**